tests/ert3/console/integration/test_cli.py

```diff
@@ -122,7 +122,7 @@ def test_cli_init_example_inside_example(tmpdir, ert_storage):
         with patch.object(sys, "argv", args):
             with pytest.raises(
                 SystemExit,
-                match="Already inside an ERT workspace",
+                match="Already inside an ERT example workspace.",
```

Comment:
We should not separate between an `example workspace` and a `workspace`. I suggest you adapt the error message and keep the test as is...